packages/manager/modules/carrier-sip/src/endpoints/index.js

```diff
@@ -0,0 +1,20 @@
+import angular from 'angular';
+
+// Peer dependencies.
+import 'angular-translate';
```

Comment:
FYI, you can
```suggestion
import angularTranslate from 'angular-translate';
```

Reply:
@FredericEspiau as it is a peer it is not indicated

Reply:
Ah, good point !

Reply:
Thanks @marie-j for your reply.